src/map.rs

```diff
@@ -476,7 +476,7 @@ impl UnseqData {
         for (key, val) in insert {
             match new_data.entry(key) {
                 Entry::Occupied(entry) => {
-                    let _ = errors.insert(entry.key().clone(), EntryError::EntryExists(0));
+                    let _ = errors.insert(entry.key().clone(), Error::EntryExists(0));
```

Comment:
Is it okay to have `let  _ =`s here? (there are a few below too).

Reply:
I wasn't looking at that in this PR. 

Seems to me like we're adding the error to a vec for later use though so should be fine